models/articles.model.js

```diff
@@ -46,3 +59,26 @@ ORDER BY articles.created_at DESC;
       return rows;
     });
 };
+
+exports.selectArticleComments = (article_id) => {
+  return db
+    .query(
+      `
+SELECT comments.comment_id,
+       comments.votes,
+       comments.created_at,
+       comments.author,
+       comments.body,
+       comments.article_id
+FROM   comments
+       JOIN articles
+         ON comments.article_id = articles.article_id
+WHERE  comments.article_id = $1
+ORDER BY comments.created_at DESC;  
+    `,
```

Comment:
Does this query require a join? How could you select all the information you need from the comments table?